packages/types/src/discord/guild.ts

```diff
@@ -681,11 +703,12 @@ export interface DiscordPrunedCount {
 
 /** https://discord.com/developers/docs/resources/guild#get-guild-vanity-url */
 // TODO: This should provably have another name for clarity to what it really rappresent
-export interface DiscordVanityUrl {
+export interface DiscordVanityUrl extends Partial<Omit<DiscordInvite, 'code'>> {
   code: string | null
   uses: number
 }
 
+// TODO: We usually don't have types for the JSON request params, we should probably remove this
```

Comment:
We already have (camelized) types for these, I think we should remove these since they're not used anywhere